### PR TITLE
Bumping apache2 cookbook version from 3.3.1 to 5.0.1

### PR DESCRIPTION
**Pull request by marcel1975**
Agreed with Karol Drążek

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# 5.1.3
```
Comment (malaskowski) [inline CHANGELOG.md]: Please put this PR into `unreleased` section.
Version `5.1.3` was not released yet.

Comment (marcel1975) [reply CHANGELOG.md]: done

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+# unreleased
+* Bumping apache cookbook version from 3.3.1 to 5.0.1
+
 # unreleased
```
Comment (malaskowski) [inline CHANGELOG.md]:
```suggestion
# unreleased
* [PR-46](https://github.com/Cognifide/aet-cookbook/pull/46) Bumping apache cookbook version from 3.3.1 to 5.0.1
```

And remove the above :)

Comment (marcel1975) [reply CHANGELOG.md]: done

**Review by malaskowski — APPROVED** [review]

**Pull request merged**